### PR TITLE
slick-flyway-postgres

### PR DESCRIPTION
**Pull request by johnolos**
I've could have added more throughly tests, but they can come later in another PR.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -6,6 +6,10 @@ val ScalatraVersion = "2.5.0"
 
 ScalatraPlugin.scalatraSettings
 
+flywayUrl := sys.env.getOrElse("DB_URL", default = "DB_URL_NOT_SPECIFIED")
```
Comment (hawkaa) [inline build.sbt:9]: Good stuff!

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+postgres {
```
Comment (hawkaa) [inline src/test/resources/application.conf]: You sure these can't inherit from the same file (in `src/resources`?)

File: src/test/scala/io/pubcrawler/stout/db/TableDefinitionsSpec.scala [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+
+class TableDefinitionsSpec extends FlatSpec with TableDefinitions with DbConnection with Matchers with BeforeAndAfter {
+
+  it should "add users" in {
+    val action = DBIO.seq(
+      users ++= userData
+    )
+    try {
+      Await.result(db.run(action), Duration.Inf)
```
Comment (hawkaa) [inline src/test/scala/io/pubcrawler/stout/db/TableDefinitionsSpec.scala]: 1 Second?

**Review by hawkaa — APPROVED** [review]
Good job, @johnolos

File: src/test/resources/application.conf (hunk 2 above)
Comment (johnolos) [reply src/test/resources/application.conf]: I removed the file entirely for now. Is that ok?

**Pull request merged**